controllers/constant/odlm.go

```diff
@@ -240,7 +245,14 @@ spec:
     packageName: db2u-operator
     scope: public
     installPlanApproval: {{ .ApprovalMode }}
-`
+  - channel: {{ .Channel }}
+    name: user-data-services-operator
+    namespace: {{ .MasterNs }}
+    packageName: user-data-services-operator-certified
+    scope: public
+    installPlanApproval: {{ .ApprovalMode }}
+    sourceName: {{ .CatalogSourceName }}
+    sourceNamespace: {{ .CatalogSourceNs }}
```

Comment:
@shikhasriva 

I believe `user-data-services` has its own catalogsource in the airgap installation. Then we don't need to add 
```
    sourceName: {{ .CatalogSourceName }}
    sourceNamespace: {{ .CatalogSourceNs }}
```

Reply:
Hi @horis233 I didn't get your comment. Why other services in registry are having catalogsource ?

Reply:
Hi @Shrivastava-Varsha 

For the other default Common Service services, they are sharing the same CatalogSource, for instance `opencloud-operator` CatalogSource.

When the code is running, the template `sourceName: {{ .CatalogSourceName }}` will be replace by `sourceName: `opencloud-operator`. We add it for convenience.

Therefore, if other services are using their own CatalogSource, we should leave it blank here. And cs-operator will look at their `PackageManifest` based on services name, then find out the CatalogSource containing those operators.

Reply:
Hi @Shrivastava-Varsha
catalog source fields are only required for the services using the operator catalog pipeline as ibm common service operator.  We add these fields for compatibility consideration. 

They are not the required fields for new onboarded services.